docs/button/button.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible"
+        content="IE=edge" />
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/f6863db2d0.js"
+        crossorigin="anonymous"></script>

```

Comment:
You can consider moving font-awesome link to your Main CSS, so you don't have to add this to all of your html files

Reply:
Thanks. Will do it.